Pages/ShowIndividualEvent.razor

```diff
@@ -0,0 +1,35 @@
+﻿@page "/event/{Id:guid}"
+@inject IFileContentProvider m_fileContentProvider
+@inject NavigationManager UriHelper
+@using System.Linq
+
+@if (charityEvent != null)
+{
+    <h1>@charityEvent.Name</h1>
+
+    <h2>@charityEvent.Description</h2>
+
+    <h3>@charityEvent.State.ToString()</h3>
+
+    <button class="btn btn-primary" @onclick="MoveToIndividualEvent">Update event</button>
+}
+else
+{
+    <h1>Sorry,but this event was not found</h1>
+}
```

Comment:
Still not sure about this, would be great to get a third opinion.

Reply:
what's wrong with this?

Reply:
Just had some doubts if it is correct to put an if at a @page section.

Reply:
Oh, never worked with blazor before so I don't know best practices. I'd say let's keep it as it is for now and ask during lecture.